<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
	<testsuite errors="0" failures="0" file="API/test_tenantRegistration.py" name="API.test_tenantRegistration.TenantRegistration-20200213221535" skipped="0" tests="5" time="0.016" timestamp="2020-02-13T22:15:36">
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="85" name="test_tenant_Email" time="0.011" timestamp="2020-02-13T22:15:36">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="34" name="test_tenant_allCorrect" time="0.003" timestamp="2020-02-13T22:15:36">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="71" name="test_tenant_noFirstName" time="0.001" timestamp="2020-02-13T22:15:36">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="56" name="test_tenant_noLastName" time="0.001" timestamp="2020-02-13T22:15:36">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantRegistration.TenantRegistration" file="API/test_tenantRegistration.py" line="98" name="test_tenant_noPassword" time="0.001" timestamp="2020-02-13T22:15:36">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="0" file="API/test_PropertyManagerRegistration.py" name="API.test_PropertyManagerRegistration.PMRegistration-20200213221535" skipped="0" tests="5" time="0.007" timestamp="2020-02-13T22:15:37">
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="36" name="test_pm_allCorrect" time="0.002" timestamp="2020-02-13T22:15:37">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="78" name="test_pm_missing_email" time="0.001" timestamp="2020-02-13T22:15:37">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="55" name="test_pm_missing_firstName" time="0.001" timestamp="2020-02-13T22:15:37">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="67" name="test_pm_missing_lastName" time="0.001" timestamp="2020-02-13T22:15:37">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistration" file="API/test_PropertyManagerRegistration.py" line="89" name="test_pm_missing_password" time="0.001" timestamp="2020-02-13T22:15:37">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="0" file="API/test_PropertyManagerRegistration.py" name="API.test_PropertyManagerRegistration.PMRegistrationRoopairs-20200213221535" skipped="0" tests="3" time="0.005" timestamp="2020-02-13T22:15:38">
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistrationRoopairs" file="API/test_PropertyManagerRegistration.py" line="157" name="test_pm_allCorrectPlusProps" time="0.003" timestamp="2020-02-13T22:15:38">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistrationRoopairs" file="API/test_PropertyManagerRegistration.py" line="139" name="test_pm_noEmail" time="0.001" timestamp="2020-02-13T22:15:38">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerRegistration.PMRegistrationRoopairs" file="API/test_PropertyManagerRegistration.py" line="148" name="test_pm_noPass" time="0.001" timestamp="2020-02-13T22:15:38">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="1" file="API/test_PropertyCRUD.py" name="API.test_PropertyCRUD.UpdateProperty-20200213221535" skipped="0" tests="5" time="0.013" timestamp="2020-02-13T22:15:38">
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="295" name="test_create_property_bad_field_bath" time="0.002" timestamp="2020-02-13T22:15:38">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="277" name="test_create_property_bad_field_city" time="0.001" timestamp="2020-02-13T22:15:38">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="314" name="test_create_property_bad_field_mult" time="0.001" timestamp="2020-02-13T22:15:38">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="259" name="test_create_property_bad_field_street" time="0.001" timestamp="2020-02-13T22:15:38">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.UpdateProperty" file="API/test_PropertyCRUD.py" line="189" name="test_update_property_allCorrect" time="0.008" timestamp="2020-02-13T22:15:38">
			<failure message="'failure' != 'success'
- failure
+ success
" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 615, in run
    testMethod()
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/test_PropertyCRUD.py", line 215, in test_update_property_allCorrect
    self.assertEqual(responseData.get(STATUS), SUCCESS)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: 'failure' != 'success'
- failure
+ success

]]>			</failure>
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="0" file="API/test_tenantLogin.py" name="API.test_tenantLogin.TenantLogin-20200213221535" skipped="0" tests="7" time="0.009" timestamp="2020-02-13T22:15:39">
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="35" name="test_tenant_allCorrect" time="0.004" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="62" name="test_tenant_incorrectEmail" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="86" name="test_tenant_incorrectEmailField" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="102" name="test_tenant_incorrectFields" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="70" name="test_tenant_incorrectPass" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="78" name="test_tenant_incorrectPassAndEmail" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_tenantLogin.TenantLogin" file="API/test_tenantLogin.py" line="94" name="test_tenant_incorrectPassField" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="1" failures="1" file="API/test_PropertyCRUD.py" name="API.test_PropertyCRUD.CreateProperty-20200213221535" skipped="0" tests="6" time="0.017" timestamp="2020-02-13T22:15:39">
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="146" name="test_create_property_bad_field_bath" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="130" name="test_create_property_bad_field_city" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="163" name="test_create_property_bad_field_mult" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="114" name="test_create_property_bad_field_street" time="0.001" timestamp="2020-02-13T22:15:39">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="82" name="test_create_property_duplicate" time="0.002" timestamp="2020-02-13T22:15:39">
			<failure message="'failure' != 'success'
- failure
+ success
" type="AssertionError">
<![CDATA[Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 615, in run
    testMethod()
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/test_PropertyCRUD.py", line 107, in test_create_property_duplicate
    self.assertEqual(responseData.get(STATUS), SUCCESS)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 680, in fail
    raise self.failureException(msg)
AssertionError: 'failure' != 'success'
- failure
+ success

]]>			</failure>
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyCRUD.CreateProperty" file="API/test_PropertyCRUD.py" line="36" name="test_create_property_allCorrect" time="0.010" timestamp="2020-02-13T22:15:39">
			<error message="Expecting value: line 3 column 1 (char 2)" type="JSONDecodeError">
<![CDATA[Traceback (most recent call last):
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/unittest/case.py", line 615, in run
    testMethod()
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/test_PropertyCRUD.py", line 47, in test_create_property_allCorrect
    responseData = getInfo(LOGIN, data)
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/helperFuncsForTesting.py", line 49, in getInfo
    response = client.post(path=reverse(endpoint), data=data, format="json")
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/test.py", line 294, in post
    path, data=data, format=format, content_type=content_type, **extra)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/test.py", line 207, in post
    return self.generic('POST', path, data, content_type, **extra)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/test.py", line 232, in generic
    method, path, data, content_type, secure, **extra)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/django/test/client.py", line 421, in generic
    return self.request(**r)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/test.py", line 283, in request
    return super().request(**kwargs)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/test.py", line 235, in request
    request = super().request(**kwargs)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/django/test/client.py", line 496, in request
    raise exc_value
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/views.py", line 505, in dispatch
    response = self.handle_exception(exc)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/views.py", line 465, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/views.py", line 476, in raise_uncaught_exception
    raise exc
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/views.py", line 502, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/aberard/Programming/HomePairs/django_env/lib/python3.7/site-packages/rest_framework/decorators.py", line 50, in handler
    return func(*args, **kwargs)
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/views.py", line 227, in login
    pmTest = pmLogin(email, password)
  File "/Users/aberard/Programming/HomePairs/2.0/homepairs/HomepairsApp/API/views.py", line 177, in pmLogin
    info = json.loads(response.text)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 3 column 1 (char 2)
]]>			</error>
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
	<testsuite errors="0" failures="0" file="API/test_PropertyManagerLogin.py" name="API.test_PropertyManagerLogin.PropertyManagerLogin-20200213221535" skipped="0" tests="9" time="0.011" timestamp="2020-02-13T22:15:47">
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="34" name="test_pm_allCorrect" time="0.002" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="70" name="test_pm_incorrectEmailField" time="0.001" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="86" name="test_pm_incorrectFields" time="0.001" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="78" name="test_pm_incorrectPassField" time="0.001" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="94" name="test_pm_propOnRoopairs" time="0.003" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="113" name="test_pm_propOnRoopairsXFour" time="0.002" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="46" name="test_pm_wrongEmail" time="0.001" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="54" name="test_pm_wrongPass" time="0.001" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
		<testcase classname="API.test_PropertyManagerLogin.PropertyManagerLogin" file="API/test_PropertyManagerLogin.py" line="62" name="test_pm_wrongPassAndEmail" time="0.001" timestamp="2020-02-13T22:15:47">
			<system-out>
<![CDATA[]]>			</system-out>
			<system-err>
<![CDATA[]]>			</system-err>
		</testcase>
	</testsuite>
</testsuites>
